schema/record-package-schema.json

```
@@ -156,12 +156,12 @@
         },
         "compiledRelease": {
           "title": "Compiled release",
-          "description": "This is the latest version of all the contracting data, it has the same schema as an open contracting release.",
+          "description": "The latest value of each field at the time of the record's publication, it has the same schema as an open contracting release.",
```

Fixed pre-existing run-on sentence. The sentence isn't needed, but I leave that to https://github.com/open-contracting/standard/issues/330

```suggestion
          "description": "The latest value of each field at the time of the record's publication. It follows the same schema as a release.",
```